### ISSUE TITLE
unable to upload media other than PNG

### ISSUE THREAD

**Issue by robindunkerley** [exiled-apes/candy-machine-mint#60]
when trying to upload an asset folder with paired GIF/JSON files, after running Metaplex upload I am receiving an error that the amount of JSON files is not corresponding with the amount of PNG files

I've tried making changes to the CLI.ts file to allow different media and remove the check on PNG and running yarn build, but it still throws the same error - so it seems to be running a cached version of CLI.js which I cannot change (?)

**Comment by robindunkerley** [comment]
I also tried deleting cache ^

**Comment by the-wuh** [comment]
I don't think you can upload any format other than .png but i think if you want you add file with any other format, you'd have to include if under the properties key in the token metadata and use a .png as the cover. But I haven't tried this.

**Comment by robindunkerley** [comment]
Ah yeah I did update the json metadata also but sing .png caused the gif to become a sequence of PNGs so I thought that wouldn't work. However,  are you suggesting that if it's uploaded that way, the altered metadata will cause it to be displayed successfully as a gif on arweave?

**Comment by the-wuh** [comment]
You may use the ```?ext={file_extension}``` query to provide information on the file type.
Try this.

**Comment by devJovanni12** [comment]
Hi. @robindunkerley 
Did you resolve this issue yet?
I tried to upload mp4 file as well, but I can not upload the mp4 without the png.
thank you in advance.
